### PR TITLE
[WEB-3813] Update implementation of `getInvoice` and replace with new implementation

### PR DESCRIPTION
**Pull request by takayuki-serato**

### CODE REVIEW

File: sws_py_sdk/ecom.py [hunk 1]
```diff
@@ -57,20 +57,24 @@ def get_subscriptions(self, subscription_id=''):
             endpoint=endpoint,
             method="GET"
         )
-        
 
-    def get_invoices(self, order_id):
-        """ Get list of all invoices for a user
+    def get_invoice(self, order_id, invoice_id, accept='application/json'):
```
Comment (Mike-Serato) [inline sws_py_sdk/ecom.py]: Will we have to update the method name wherever this sdk is implemeted?

File: sws_py_sdk/ecom.py [hunk 2]
```diff
@@ -57,20 +57,24 @@ def get_subscriptions(self, subscription_id=''):
...
+                ID of the order
+            invoice_id: int
+                ID of the invoice
+            accept: string
+                Accept header, can be JSON, PDF or HTML
         """
-        endpoint = "/api/v1/me" if self.sws.user_id == 0 else "/api/v1/users/" + str(self.sws.user_id)
-        endpoint = endpoint + "/orders/" + str(order_id) + "/invoice" 
+        prefix = "/api/v1/me" if self.sws.user_id == 0 else "/api/v1/users/" + str(self.sws.user_id)
+        endpoint = f"{prefix}/orders/{order_id}/invoices/{invoice_id}"
```
Comment (Mike-Serato) [inline sws_py_sdk/ecom.py]: Is that 'f' there a typo?

File: sws_py_sdk/ecom.py (hunk 1 above)
Comment (takayuki-serato) [reply sws_py_sdk/ecom.py]: Yes, we have to update the usages of this function either way - the old method will no longer work (and the endpoint it was hitting will also be removed).

File: sws_py_sdk/ecom.py (hunk 2 above)
Comment (takayuki-serato) [reply sws_py_sdk/ecom.py]: This is a python feature, similar to template strings in JavaScript. `f` allows you to create an interpolatable string.

**Pull request merged**